lib/rules/media-feature-range-notation/index.mjs

```diff
@@ -94,6 +94,8 @@ const rule = (primary) => {
 						if (expectedMediaQueryList === atRule.params) return;
 
 						atRule.params = expectedMediaQueryList;
+
+						return atRule;
```

Comment:
This is the change most rule authors will need to make to ensure compatibility with this new feature.

They only need to return a PostCSS node that encloses all changes in their `fix()` callback.

Reply:
If it's not too perf heavy, this looks like a very elegant solution.

Reply:
I've since updated this to a separate field `fixedNode` instead of a return value on `fix()`. This is more in line with existing values.